plugins/pulp_docker/plugins/importers/upload.py

```diff
@@ -337,16 +337,14 @@ def process_main(self, item=None):
         :return:     None
         """
         if isinstance(item, models.Blob):
-            blob_src_path = os.path.join(self.get_working_dir(), item.digest.split(':')[1] + '.tar')
+            checksum_type, _, checksum = item.digest.rpartition(':')
+            if not checksum_type:
```

Comment:
why did you introduce this check? especially if you don't use it further anywhere else?

Reply:
It's not being used now, but see my other PR. I can back that off and move it into the other PR, if you prefer.

Reply:
... the other PR which I haven't initiated yet. Preview is here:

https://github.com/mibanescu/pulp_docker/commit/33e86f60d13a891d847c36235f400826dba556bb

Reply:
yes please, move it away for now. let's introduce stuff as it is needed and used